topo_processor/data/data_transformers/data_transformer_repo.py

```diff
@@ -21,8 +21,15 @@ async def transform_data(self, item: Item) -> None:
             for transformers in self.transformers:
                 if transformers.is_applicable(item):
                     start_time = time_in_ms()
-                    await transformers.transform_data(item)
+                    try:
+                        await transformers.transform_data(item)
+                    except Exception as error_msg:
+                        item.is_valid = False
```

Comment:
you should return here, no point continuing.

Reply:
done